src/devtools/client/webconsole/reducers/messages.js

```diff
@@ -1066,6 +1066,21 @@ function getDefaultFiltersCounter() {
   return count;
 }
 
+//get the point for the corresponding command, regardless of where we're currently paused
+function getPausePoint(newMessage, state) {
+  if (
+    newMessage.type === constants.MESSAGE_TYPE.RESULT &&
+    newMessage.parameters &&
+    newMessage.parameters.length > 0 &&
+    newMessage.parameters[0]._pause &&
+    newMessage.parameters[0]._pause.point
+  ) {
```

Comment:
we can clean this up even further with  optional chaining.

```js
newMessage.parameters.[0]?._pause.point
```

we could add more `?` but i don't think we need them...

```js
newMessage?.parameters?.[0]?._pause?.point
```

why not:

* parameters should always be an array
*  if there is a _pause there should be a point

last tweak, it makes me uncomfortable to access a private property `_pause`. Perhaps we should add an accessor

```js
newMessage.parameters.[0]?.getExecutionPoint()
```